### PR TITLE
Adding a bug on front-end to show api error messages on facility search

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to?
-  n/a

#### Brief feature description
-  to verify this story - https://osi-cwds.atlassian.net/secure/RapidBoard.jspa?rapidView=117&modal=detail&selectedIssue=CALS-5837

#### Does this PR block/effect other stories?
-  no

#### Do any commands need to be run in order to run the project with this PR?
-  no

#### Code coverage percent diff as a result of this PR?
- n/a

### CODE REVIEW

File: app/javascript/components/common/nameCommonFields.jsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const NameCommonFields = ({
         gridClassName='col-md-4'
         id={idPrefix + 'first_name'}
         value={firstName}
-        maxLength='21'
+        maxLength='20'
```
Comment (Nasir2712) [inline app/javascript/components/common/nameCommonFields.jsx]: just fixing it back to how it was

**Comment by Nasir2712** [comment]
needs to be merged and deployed so that I can show 500 errors on search.

**Comment by sunilb0575** [comment]
jenkins, retest this please

**Comment by sunilb0575** [comment]
jenkins, retest this please

**Comment by sunilb0575** [comment]
jenkins retest this please

**Comment by Nasir2712** [comment]
jenkins retest this please

**Comment by Nasir2712** [comment]
jenkins, retest this please

**Comment by Nasir2712** [comment]
jenkins, retest this please

**Pull request merged**